Formula/unison.rb

```diff
@@ -31,6 +31,10 @@ def install
     ENV.delete "NAME" # https://github.com/Homebrew/homebrew/issues/28642
     system "make", "UISTYLE=text"
     bin.install "src/unison"
+    if OS.linux?
+      # unison-fsmonitor is built just for Linux targets
+      bin.install "src/unison-fsmonitor"
+    end
```

Comment:
```suggestion
    # unison-fsmonitor is built just for Linux targets
    bin.install "src/unison-fsmonitor" if OS.linux?
```
Please squash this change into your existing commit. Thanks!